tests/e2e/utils.js

```diff
@@ -143,21 +144,34 @@ export const isBlockInsertedInWidgetsArea = async ( blockName ) => {
  *
  * There are two different possible site editor pages:
  *
- * 1. `themes.php?page=gutenberg-edit-site` is the one used and available if the Gutenberg plugin is enabled.
- * 2. `site-editor.php` is the one available in WP Core.
+ * 1. `themes.php?page=gutenberg-edit-site` this is legacy editor access used for wp <=5.8 compatibility
+ * 	  themes.php is not accessible in 5.9+ when local Gutenberg is active.
```

Comment:
Missing some backticks for consistency around `themes.php`. But, more importantly, I am not sure I understand this comment, could you clarify it a bit?

So `themes.php?page=gutenberg-edit-site` seems to be the way to go whenever Gutenberg is installed on <6.0, right? At least that's what I get by reading your code. The comment says `themes` is not accessible in 5.9+ when local Gutenberg is active, and I'm not sure I understand.

Reply:
Argh, was tweaking text and made error - should be site-editor is not accessible! 

Will reword it :)